idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -184,7 +203,10 @@ export default PageView.extend({
         { value: 'dislike', title: 'I don\'t like this feature.' },
         { value: 'notuseful', title: 'This isn\'t useful for me.' },
         { value: 'other', title: 'Something else.' }
-      ]
+      ],
+      uninstall: this.uninstall,
```

Comment:
Making the feedback view also uninstall the addon is giving it too many responsibilities. It's also reused for the other feedback case. If we definitely want to hold off the uninstall until an action is taken in the feedback form, it would be better to just add a generic callback hook when done. Then, perform the uninstall in a callback here